### PR TITLE
add algolia search

### PR DESCRIPTION
**Pull request by KieranHayes**

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -12,7 +12,11 @@ description: dkdeploy documentation
 baseurl: "" # the subpath of your site, e.g. /blog
 url: "https://dkdeploy.github.io" # the base hostname & protocol for your site
 twitter_username: dkd_de
-github_username:  dkdeploy
+github_username:  KieranHayes
```
Comment (klyonrad) [inline _config.yml]: why?

File: Gemfile [hunk 2]
```diff
@@ -1,2 +1,5 @@
 source 'https://rubygems.org'
 gem 'github-pages', group: :jekyll_plugins
+group :jekyll_plugins do
+  gem 'algoliasearch-jekyll', '~> 0.8.0'
+end
```
Comment (websi) [inline Gemfile]: No newline at end of file

File: _config.yml (hunk 1 above)
Comment (KieranHayes) [reply _config.yml]: Oops

**Comment by klyonrad** [comment]
please squash Commit b46acbb34792f5b88145e7162d64216ff1cab9c6 into another Commit

File: _config.yml [hunk 3]
```diff
@@ -16,3 +16,7 @@ github_username:  dkdeploy
 exclude: [foundation, vendor]
 # Build settings
 markdown: kramdown
+algolia:
+  application_id: 'QOFNLOOCWH'
+  index_name:     'dkdeploy_dev'
+  record_css_selector: 'p,ul,td,th,h3,code'
```
Comment (websi) [inline _config.yml:22]: No newline at end of file

File: _config.yml [hunk 4]
```diff
@@ -16,3 +16,7 @@ github_username:  dkdeploy
 exclude: [foundation, vendor]
 # Build settings
 markdown: kramdown
+algolia:
+  application_id: 'QOFNLOOCWH'
+  index_name:     'dkdeploy_dev'
```
Comment (websi) [inline _config.yml]: dev?

File: _includes/footer.html [hunk 5]
```diff
@@ -2,18 +2,50 @@
...
-        <p>&copy 2016 <a href="https:/www.dkd.de/">dkd Internet Service GmbH</a> <i class="fi-heart animated"></i> <a href="mailto:opensource@dkd.de">opensource@dkd.de</a></p>
+        <p>&copy; 2016 <a href="https://www.dkd.de/">dkd Internet Service GmbH</a> <i class="fi-heart animated"></i> <a href="mailto:opensource@dkd.de">opensource@dkd.de</a></p>
       </div>
     </div>
   </div>
 </footer>
+<script>
+  window.ALGOLIA_CONFIG = {
+    'applicationId': 'QOFNLOOCWH',
+    'indexName': 'dkdeploy_dev',
```
Comment (websi) [inline _includes/footer.html]: dev?

Comment (KieranHayes) [reply _includes/footer.html]: changed name to dkdeploy_production

**Pull request closed without merging**